crates/oxc_transformer/src/proposals/explicit_resource_management.rs

```diff
@@ -107,6 +113,13 @@ impl<'a> Traverse<'a> for ExplicitResourceManagement<'a, '_> {
             false,
         ));
 
+        let scope_id = match for_of_stmt.body {
+            Statement::BlockStatement(ref mut block) => block.scope_id(),
+            _ => ctx.create_child_scope(for_of_stmt_scope_id, ScopeFlags::empty()),
+        };
```

Comment:
Purely a code style thing, but would you mind using this style instead? This is our convention throughout the transformer:

```suggestion
        let scope_id = match &mut for_of_stmt.body {
            Statement::BlockStatement(block) => block.scope_id(),
            _ => ctx.create_child_scope(for_of_stmt_scope_id, ScopeFlags::empty()),
        };
```